src/stock-physical-inventory-draft/messages_en.json

```diff
@@ -12,7 +12,7 @@
   "stockPhysicalInventoryDraft.productWithDisplayUnit": "${fullProductName} - ${displayUnit}",
   "stockPhysicalInventoryDraft.productCode": "Product Code",
   "stockPhysicalInventoryDraft.lotCode": "Lot Code",
-  "stockPhysicalInventoryDraft.lotCodeMustBeUnique": "Lot code must be unique",
+  "stockPhysicalInventoryDraft.lotCodeMustBeUnique": "Trade Item is required to add a Lot Code",
```

Comment:
This seems to not resolve the main issue. I suppose that if some lot code is already used for this product new message will be displayed what is wrong. This message should only be shown when backend return missing tradeitem.